cmd/ls.go

```diff
@@ -22,7 +22,7 @@ import (
 	"text/tabwriter"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
goimports is a linter to handle imports for golang and enforce good practice for imports.
When I run goimports on ls.go it suggests the following diff:

goimports -d ls.go                                                                                                                                                                                                    ⏎
diff ls.go gofmt/ls.go
--- /var/folders/xr/zvgg6czx3kngbz4wr8llz20m0000gn/T/gofmt307211967	2016-07-16 20:58:55.000000000 -0700
+++ /var/folders/xr/zvgg6czx3kngbz4wr8llz20m0000gn/T/gofmt120580370	2016-07-16 20:58:55.000000000 -0700
@@ -22,6 +22,7 @@
 	"text/tabwriter"

 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
+	"github.com/dustin/go-humanize"
 	"github.com/spf13/cobra"
 )

The real issue here is that the package "humanize" is stored in a folder named "go-humanize"  going against the good practice for package naming.

That's why I named the import to make it clearer that everything in go-humanize is actually in the humanize package.

If you want me to drop this patch, I will remove it and resend the PR. What do you think? Does it make things clearer?